deployments/bridges/westend-millau/dashboard/grafana/relay-westend-to-millau-headers-dashboard.json

```diff
@@ -0,0 +1,694 @@
+{
```

Comment:
I'm assuming this is just a copy of existing dashboard of other networks, right?

Reply:
That's correct